### DISCUSSION TITLE
title bar theme change

### DISCUSSION THREAD

**Discussion by OnlineLearningTutorials** [linuxmint/discussions#586] [Ideas]
As I used previous version of linuxmint there is extra option to choose title bar theme. But for recent release like linuxmint22 that option is removed with introducing modern style of theming. I like [whitesur](https://github.com/linuxmint/cinnamon-spices-themes/blob/master/WhiteSur-Dark-solid/screenshot.png) style theming. So is there a way to change the theme of title bar or there is some technical limitation.

**Comment by Secret-chest** [comment]
There is a technical limitation: the title bars were previously handled by the window manager itself, now they are handled by the widget toolkit GTK+ which can only use one theme per window.
